narayana-full/pom.xml

```diff
@@ -356,6 +356,7 @@
                     -exclude com.arjuna.orbportability.internal -exclude com.arjuna.ats.jbossatx
                     -exclude com.arjuna.mw.wstx -exclude org.jboss.jbossts.txbridge
                     -exclude org.jboss.jbossts.xts -exclude org.jboss.jbossts.xts11
+                    -exclude com.arjuna.ats.internal.arjuna.objectstore.slot
```

Comment:
Do you know why the exclude of com.arjuna.ats.internal on line 355 is not enough? perhaps we need to somehow tell Javadoc exclude recursively. If that is not available as an option this works for me

Reply:
I don't think this line actually had any effect. The main change was to remove the provided scope from the dependency.

Reply:
To test this, how about a commit reverting that line to see if it still passes?

Reply:
I think try a commit without the other change rather than this, we don't want class in the javadocs so probably the exclusion here is what we need and if so then although I don't think we should need to add each package independently perhaps we do for now? We should try to get this excluded recursively for all internal sub-packages though.